insertion_sort/C/insertionsort.c

```diff
@@ -0,0 +1,30 @@
+// Program implementing insertion sort using array in C
+#include<stdio.h>
+// Function to sort an array using insertion sort
+void insertionsort(int ar_size,int *ar) {
+    int temp, j;
+    for(int i = 1; i < ar_size; i++) {
```

Comment:
Missing space before ( in for(  [whitespace/parens] [5]

*Origin: CPPLintBear, Section: `all.cpp`.*